cid/common.py

```diff
@@ -1287,8 +1287,9 @@ def create_or_update_dataset(self, dataset_definition: dict, dataset_id: str=Non
             else:
                 print(f'No update requested for dataset {compiled_dataset.get("DataSetId")} {compiled_dataset.get("Name")}={found_dataset.name} ')
         else:
-            self.qs.create_dataset(compiled_dataset)
```

Comment:
Update dataset also requires update of schedule

Reply:
Current premise is, if there is existing schedule, do nothing. 
For the time being, I have added that if dataset is updated, create refresh schedule if it doesn't exist.